static/templates/settings/demo_organization_warning.hbs

```diff
@@ -0,0 +1,12 @@
+{{#if is_demo_organization }}
+<div class='demo-organization-warning'>
+    {{#tr}}
+    This demo organization will be automatically deleted in {days_remaining} days.&nbsp;
+    {{/tr}}
+    {{#if is_owner }}
+        {{#tr}}
+        Convert to make permanent.
+        {{/tr}}
```

Comment:
Maybe "Convert to make it permanent." would be more clear?

Reply:
That's definitely better. Another option I thought of is "Make it a permanent organization". See two screenshots below.

<details>
<summary>Alya's updated text</summary>

![Screenshot from 2022-08-23 16-10-45](https://user-images.githubusercontent.com/63245456/186180987-be5a7a7d-0718-4516-802e-3292e44f89ca.png)
</details>

<details>
<summary>Lauryn's alternative text</summary>

![Screenshot from 2022-08-23 16-10-13](https://user-images.githubusercontent.com/63245456/186181112-f9413001-93e4-49b2-9779-c975ab1d70a7.png)
</details>

Reply:
Hm, I think I prefer keeping "convert" as the action verb here (and elsewhere). @timabbott what do you think?

Reply:
With the current updates, the text is "Convert to make it permanent".